packages/viz/src/App.js

```diff
@@ -5,7 +5,7 @@ import * as mergePolicy from './merge-deep.js'
 import Nav from './views/nav.js'
 
 
-const { DepGraph } = require('./graphs/DepGraph')
+const { DepGraph } = require('./graphs/DepGraph.js')
```

Comment:
This line is weird... 
@kumavis why is there a mix of import and require in one file?

Reply:
Indeed. Do we want to consider enabling `no-var-requires`/`no-require-imports` as well?

Reply:
haha idk! viz is getting old and dusty